src/utils/Experiment.utils.js

```diff
@@ -0,0 +1,186 @@
+/* global Experiments, Projects */
+
+/**
+ * Delete experiment and reorganizes the experiment list
+ *
+ * @param {object[]} experiments experiments list
+ * @param {string} experimentId deleted experiment id
+ * @returns {object[]} new experiment list
+ */
+export const deleteExperiment = (experiments, experimentId) => {
+  // ! NOT A CLONE = they are in the same memory address = MODIFYING PARAM
+  const experimentsAux = experiments;
```

Comment:
Será se faz sentido ser um clone?

Reply:
Sim, esse comentário acima foi eu que fiz nesse mesmo PR, mas não mexi nisso para não correr o risco de quebrar nada. Pode ser que não esteja causando nenhum erro se essa função for usada dentro de uma action. Se for uma variável de estado pode causar erros inesperados.

Mas em todo caso acho que daria pra fazer outro PR pra corrigir esses detalhes e até simplificar mais algumas funções em /utils que tem muitos IFs dentro. O que acha? :thinking:

Reply:
Acho que é uma boa mesmo, em outro focar na otimização e diminuir complexidade.